en/docs/tutorials/configure-automatic-deployment.md

```diff
@@ -0,0 +1,105 @@
+# Configure Automatic Deployment
+
+When you develop a Choreo component, you need to commit its source code into a GitHub repository. You can select a repository that Choreo manages or a repository that you own and manage.
+
+Choreo-managed repositories are not accessible to external users. Therefore, when multiple developers need to develop components in collaboration, they should connect their own repositories to save the source code. In such organizations, deploying a component manually each time a developer updates the component design can be cumbersome. Therefore, Choreo allows you to configure automatic deployment for your components. When you enable automatic deployment, every commit that changes the component implementation automatically deploys the component to the development environment and triggers a build.
```

Comment:
do we need the word "own"? Can we rephrase it to "Therefore, when multiple developers need to develop components in collaboration, they should connect their repositories to save the source code."

Reply:
Can we rephrase this to "In such organizations, manually deploying a component each time a developer updates the component design can be cumbersome"

Reply:
is it every commit? or every commit with a diff specifically?

Reply:
Rewrote this paragraph.